### DISCUSSION TITLE
[Feat]: Development based on code server

### DISCUSSION THREAD

**Discussion by chenhuanguang** [coder/code-server#5672] [Q&A]
Whether the source code of the code server package provided is the code of the current code base. I want to modify the code server page. I think the source code should be modified, but after the source code is modified, it should be compiled, but I don't know how to do it, and then make a package

**Comment by chenhuanguang** [comment]
Would you please tell me in detail? Thank you very much

**Reply by jsjoeio** [reply]
You can find this information on https://coder.com/docs/code-server/latest/CONTRIBUTING
